pkg/pipeline/WorkflowDagExecutor.go

```diff
@@ -1053,7 +1054,7 @@ func (impl *WorkflowDagExecutorImpl) TriggerDeployment(cdWf *pipelineConfig.CdWo
 			UpdatedOn: time.Now(),
 		},
 	}
-	err = impl.pipelineStatusTimelineService.SaveTimeline(timeline, nil)
+	err = impl.pipelineStatusTimelineService.SaveTimeline(timeline, nil, false)
```

Comment:
create local variable isAppStore, and pass that variable

Reply:
okay done